EDMCLogging.py

```diff
@@ -91,6 +92,9 @@
     **kwargs
 )
 
+# make logging use UTC for times
```

Comment:
"... because we want to be able to more easily match our log files with e.g. game journals, which are always in ed-server's idea of UTC"